ProcessOptimizer/model_systems/color_pH.py

```diff
@@ -0,0 +1,339 @@
+import csv
+import math
+import numpy as np
+import os
+
+from typing import List
+from .model_system import ModelSystem
```

Comment:
```suggestion
from ProcessOptimizer.model_systems.model_system import ModelSystem
```
Local import doesn't work on my VSCode, but it might just be an issue with my install...

But I can't simply do `from ProcessOptimizer import ModelSystem`, because ModelSystem is not imported into `ProcessOptimizer/__init__.py`. We need to fix that.